akkaserverless/akkaserverless_service.py

```diff
@@ -0,0 +1,123 @@
+"""
+Copyright 2020 Lightbend Inc.
+Licensed under the Apache License, Version 2.0.
+"""
+import logging
+import multiprocessing
+import os
+from concurrent import futures
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+import grpc
+
+from akkaserverless.akkaserverless.component.action.action_pb2_grpc import add_ActionsServicer_to_server
+from akkaserverless.action_protocol_entity import Action
+from akkaserverless.action_servicer import AkkaServerlessActionProtocolServicer
+from akkaserverless.discovery_servicer import AkkaServerlessEntityDiscoveryServicer
+from akkaserverless.akkaserverless.protocol.discovery_pb2_grpc import add_DiscoveryServicer_to_server
+from akkaserverless.event_sourced_entity import EventSourcedEntity
+from akkaserverless.akkaserverless.component.eventsourcedentity.event_sourced_entity_pb2_grpc import add_EventSourcedEntitiesServicer_to_server
+from akkaserverless.eventsourced_servicer import AkkaServerlessEventSourcedServicer
+
+from akkaserverless.view import View
+from akkaserverless.value_entity import ValueEntity
+from akkaserverless.akkaserverless.component.valueentity.value_entity_pb2_grpc import add_ValueEntitiesServicer_to_server
+from akkaserverless.value_servicer import AkkaServerlessValueServicer
+
+# from grpc_reflection.v1alpha import reflection
+
+
+@dataclass
+class AkkaServerlessService:
+    logging.basicConfig(
+        format="%(asctime)s - %(filename)s - %(levelname)s: %(message)s",
+        level=logging.DEBUG,
+    )
+    logging.root.setLevel(logging.DEBUG)
+
+    __address: str = ""
+    __host = "0.0.0.0"
+    __port = "8080"
+    __workers = multiprocessing.cpu_count()
+    __event_sourced_entities: List[EventSourcedEntity] = field(default_factory=list)
+    __value_entities: List[ValueEntity] = field(default_factory=list)
+    __action_protocol_entities: List[Action] = field(default_factory=list)
+    __views: List[View] = field(default_factory=list)
+
+    def host(self, address: str):
+        """Set the address of the network Host.
+        Default Address is 127.0.0.1.
```

Comment:
on line https://github.com/cloudstateio/python-support/pull/47/files#diff-296ae8741e7ca1d6af23a82352bc2fa1814a916d2841f3c30179550c52b1b4c9R40 it defaults to 0.0.0.0.